orca-peering/README.md

```diff
@@ -0,0 +1,113 @@
+# Orca Peering
+
+This is an semi-experimental approach to solve the problem of having multiple `orca` installations (each with their own database) communicate changes with each other, for instance in a multi-region Spinnaker installation or during a database migration.
+
+The peering mechanism accomplishes a few things:
+1. (complete) Peer executions (pipelines and orchestrations) from one database (region) to another database (region)
+2. (in-progress) Allows for an `orca` operating with one database to perform actions on an execution being executed by an `orca` operating with a different database
+3. (still to come) Take ownership and resume an execution previously operated on by an `orca` with a different database
```

Comment:
```suggestion
3. (still to come) The ability to migrate incomplete executions across partitions
```

Reply:
Not sure if these are good suggestions due to some terminology confusion. Might help to define Peer, define Partition, and explain how the concepts relate.

Reply:
good point, doing now!